main.rb

```diff
@@ -1,22 +1,5 @@
 require './app'
```

Comment:
- [ ] [REQUIRED] Good job so far! The `main.rb` file should lounch the `project` when run `ruby ./main.rb` command but it isn't working, please fix the issue and check it is working or not 🙏🏼 

![image](https://github.com/aima98/OOP-School-Library1/assets/63915024/f1e6251b-ea68-4a1d-98cc-1323d196c73e)